README.md

```diff
@@ -533,9 +533,10 @@ you proceed to [step 4](#verify-itts-client-version).
 	```
 
 3. Attest with the [Intel Tiber Trust Service](https://www.intel.com/content/www/us/en/security/trust-authority.html).
-   1. Obtain an API key following this [tutorial](https://docs.trustauthority.intel.com/main/articles/tutorial-api-key.html?tabs=attestation-api-key-portal%2Cattestation-sgx-client).
+   1. Subscribe to the Intel Tiber Trust Service [free trail](https://www.intel.com/content/www/us/en/security/trust-authority.html).
```

Comment:
```suggestion
   1. Subscribe to the Intel Tiber Trust Service [free trial](https://plan.seek.intel.com/2023_ITATrialForm).
```

Reply:
I thought about doing that, but I am worried about the "2023" form and that it might change.